earth_moving/clutter/scratchpad.ipynb

```diff
@@ -62,22 +62,7 @@
       "started thread 0 \n",
```

Comment:
I think it's about time to extract what you have now in this scratchpad to a `runner.py` module.
It's becoming now our "official" entry point (not just something we try in "clutter").
Also, as you can see, tracking Jupyter notebooks in Git is a pain.
We can still use the notebooks for development purposes, jointly with the `runner.py`

Reply:
So the `runner.py` defines the standard experiment to be run? And the specific characteristics are fetched from the configuration file?